definitions/ext-rfscanner/definition.yml

```diff
@@ -0,0 +1,53 @@
+#  [mandatory] The domain of an entity, default to EXT unless otherwise indicated by the maintainers.
+domain: EXT
+# [mandatory] The type of an entity. Must be unique within its domain, but may be duplicated between domains.
+# Please add a brief description of what your type is here on the definition file.
+type: RF-Scanner
```

Comment:
```suggestion
type: RF_SCANNER
```

Reply:
Also, the directory should be consistent with the type here.  E.g. the directory now is ext-rfscanner, so the type her should be RFSCANNER (or, if the type is RF_SCANNER, then the directory should be ext-rf_scanner)